### ISSUE TITLE
Remove research page

### ISSUE THREAD

**Issue by yanokwa** [opendatakit/website#104]
I think it's easier and more thorough to find all the papers about ODK through [traditional sources](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=%22open+data+kit%22+or+%22opendatakit%22+or+%22ODK%22&btnG=) so I'd propose removing the research page.

**Comment by huangyz0918** [comment]
@yanokwa As my understanding, we should remove the `research` page, redirecting to the [google scholar page](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=%22open+data+kit%22+or+%22opendatakit%22+or+%22ODK%22&btnG=)?

<img width="1552" alt="2018-12-12 1 57 35" src="https://user-images.githubusercontent.com/15646062/49849852-4da20700-fe16-11e8-8010-ce2c622c90bc.png">

**Comment by yanokwa** [comment]
@huangyz0918 Thanks for sending in the PR. Let's have a quick discussion with @wbrunette about the general approach.

@wbrunette, I was thinking that perhaps the best way to handle this is.
1. Use the s3_website.yml file to do a redirect from /research to Google Scholar. I think the best query is `"opendatakit" OR "Open Data Kit"` (case matters)
2. We remove the research link from the community page (we don't have any other external links)
3. We remove any of the local PDFs that remain.

**Comment by wbrunette** [comment]
Would it make sense to migrate them to the history page? Why keep the deployments there and not the papers that talk about deployments? Or articles which are essentially like research?

**Comment by huangyz0918** [comment]
> Would it make sense to migrate them to the history page

+1

**Comment by yanokwa** [comment]
I don't think it makes sense to move them to history since the press and and deployments page have been flagged for removal as well.

https://github.com/opendatakit/website/issues/102
https://github.com/opendatakit/website/issues/101

**Comment by huangyz0918** [comment]
Let me update the PR as your @yanokwa proposed, and see how it looks like.

> 1. Use the s3_website.yml file to do a redirect from /research to Google Scholar. I think the best query is "opendatakit" OR "Open Data Kit" (case matters)
> 2. We remove the research link from the community page (we don't have any other external links)
> 3. We remove any of the local PDFs that remain.
